src/HabboParser.php

```diff
@@ -231,11 +231,11 @@ public static function throwHabboApiException($data)
         // Check if data is JSON
         if ($data[0] == "{") { // Quick 'hack' to see if this could be JSON
             $json = json_decode($data, true);
+            $defaultMessage = $json['errors'][0]['msg'];
```

Comment:
You'd need to move the $defaultMessage within the isset() check, because $json might not contain the key "errors". Otherwise sounds like a good plan.